benchmarks/core/arit.py

```diff
@@ -27,6 +27,12 @@ def time_Add_xy(self):
     def time_Add_xyz(self):
         Add(*[x, y, z])
 
+    def time_Add_thousands(self):
+        # from https://github.com/sympy/sympy/pull/27254
+        a = symbols("a0:2000")
+        b = Add(*a);
```

Comment:
Thanks for adding this @bjodah. Shall we move these two lines to the top so that it doesn't end up in the timing results?

Reply:
yes, sounds good.